sql/search_derived/mobile_search_clients_daily_v1/query.sql

```diff
@@ -181,5 +206,6 @@ GROUP BY
   submission_date,
   client_id,
   engine,
+  normalized_engine,
```

Comment:
Surely we don't need to group by both engine and normalized_engine? How does Ben feel about grouping by just the latter?

Reply:
We need to group by `engine` for granularity but for `normalized_engine` engine I don't think it would matter since each engine would have one normalized engine value.  I just found grouping by `normalized_engine` to be the simplest way to do it since otherwise we need to aggregate it or make this a CTE and write another small select statement.

e.g.
```sql
SELECT *, udf_strict_normalize_search_engine(engine) AS normalized_engine
FROM scd
```

Although after writing this out I actually prefer this way because it removes the repeated logic of filtering out high search count and getting the engine from the normalized search key.